packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+// solhint-disable quotes
+
+import {GovernanceMessage} from "@nomad-xyz/contracts-core/contracts/governance/GovernanceMessage.sol";
+import {TypeCasts} from "@nomad-xyz/contracts-core/contracts/XAppConnectionManager.sol";
+
```

Comment:
whitespace - prefer to delete or replace with comment

Reply:
meh, I want the conceptual separation

Reply:
when there's conceptual separation i like to indicate it with a comment, rather than leaving it implicit. achieves the separation and more clarity.

Reply:
but if you insist, the whitespace is not a blocker